ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml

```diff
@@ -1944,6 +1944,15 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-ibmcloud-ipi-dedicatedhost-static
+- as: ibmcloud-ipi-minimal-permission-f28
```

Comment:
need destructive testing too.

Reply:
need ipi-minimal-permission and ipi-minimal-permission-destructive both in all versions?

Reply:
yes, or you can merge the `minimal-permission` onto some other existing ci jobs.